src/preview.jsx

```diff
@@ -297,6 +298,17 @@ export default class Preview extends React.Component {
     return this.props.renderEditForm(formElementEditProps);
   }
 
+  showHeaderButtons() {
+    return (
+      <div className='react-form-builder-header'>
+        <div className='react-form-builder-header__inner'>
+          {this.props.onPreviewButton && <button className='button button--primary' onClick={this.props.onPreviewButton}>Preview</button>}
+          {this.props.onPublishButton && <button className='button button--primary' onClick={this.props.onPublishButton}>Publish</button>}
+        </div>
+      </div>
+    );
+  }
+
```

Comment:
are these based on the existing styles?  or do they depend on spoton lib styles?

Reply:
this is custom buttons.
these buttons' styles are copy/paste from spoton-lib component Button